File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/service/ErdOnlineConnectorService.java
package org.javamaster.invocationlab.admin.service;

import org.javamaster.invocationlab.admin.model.erd.TokenVo;
import com.alibaba.fastjson.JSONObject;

/**
 * @author yudong
 */
public interface ErdOnlineConnectorService {
    String pingDb(JSONObject jsonObjectReq, TokenVo tokenVo) throws Exception;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/service/ErdOnlineConnectorService.kt
package org.javamaster.invocationlab.admin.service

import org.javamaster.invocationlab.admin.model.erd.TokenVo
import com.alibaba.fastjson.JSONObject

/**
 * @author yudong
 */
interface ErdOnlineConnectorService {
    fun pingDb(jsonObjectReq: JSONObject, tokenVo: TokenVo): String
}
